ctc-uci/cell-dogs-frontend
# 59 create toasts and avatar upload

Authors: Michelle Lin & Lianne Lee

### What does this PR contain?
- toasts for add and delete user
- upload avatar for user, facility, and dog

### How did you test these changes?
- Add a user and verify green toast appears
- Remove a user and verify blue toast appears
- upload picture from file system and verify it appears as the new avatar

### Attach images (if applicable)
Remove User Toast
![removeUser](https://user-images.githubusercontent.com/90677583/226075408-18d78bd8-1abe-4614-a588-4ac91991f629.png)

Add User Toast
![addUser](https://user-images.githubusercontent.com/90677583/226075444-31fd9771-325d-4a1c-9340-fd499a9bd78b.png)

Upload Avatar
![2023-03-17 (3)](https://user-images.githubusercontent.com/90677583/226075519-12b69383-e0bf-4c13-8358-1284d643c571.png)
